.github/workflows/lint_python.yml

```
@@ -0,0 +1,18 @@
+name: lint_python
+on:
+  pull_request:
+  push:
+  #  branches: [master]
```

I would suggest to run linting on each pull request so that we can ensure that all code merged to `master` will be high quality in which case we won't need to trigger on push. I would suggest to replace the triggers with the following:


```suggestion
on:
  pull_request:
    types:
      - opened
      - reopened
      - synchronize
```

---

Maintainers can push directly to master and it would be unfortunate to not let them know that they had broken the build.

---

As one of the new maintainers of this project, I can commit to not pushing directly to master so I don't foresee this being an issue. Pushing directly to master for a widely used project is an anti-pattern.